coordinator/src/routes/tracker.ts

```diff
@@ -0,0 +1,58 @@
+import Hapi from "@hapi/hapi"
+import {fhir} from "@models"
+import {BASE_PATH, ContentTypes} from "./util"
+
+interface lineItem {
```

Comment:
Remove? This isn't part of a FHIR Task. Line item statuses will be returned by the GET /MedicationRequest interaction.